codejail/exceptions.py

```diff
@@ -0,0 +1,30 @@
+"""
+Codejail exceptions.
+"""
+
+
+class CodeJailBaseException(Exception):
+    """
+    Top of the hierarchy of codejail exceptions
+    """
+    pass
+
+
+class JailError(CodeJailBaseException):
+    """
+    There was a problem configuring a codejail or accessing a configured codejail.
+    """
+    pass
+
+
+class SafeExecException(Exception):
```

Comment:
Also, if the "inside-vs-outside of `exec`" changes we discussed get made, it may be useful to have differing exception types